google-newtab/css/main.css

```diff
@@ -0,0 +1,13 @@
+body{
+    display: inline-block;
+    align-content: center;
```

Comment:
align content is definitely a flexbox only thing. You can also use the * selector instead of selecting the body tag.